config/config.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 RedHat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+const (
+	OperatorConfigMapName string = "rbac-permissions-operator"
+	OperatorName          string = "rbac-permissions-operator"
```

Comment:
Are we going away from the standard of prefixing SRE/administrative namespaces with `openshift-`?

Priorly it was used as an indicator that the namespace shouldn't receive extended permissions for customers.

Reply:
This was a copy/paste from Lisa. I'm not sure what the standard is here

Reply:
Private link ,sorry, but here's the standard: https://docs.google.com/document/d/112tjemWdUxTkJw47368S4bLBVY8L1PTWFxUnQXY84uc/edit#heading=h.7k3cd18bzadj

```
The project/namespace where the operator and its components reside should be named (using the same example from above) openshift-backup-operator. The openshift- prefix is necessary because reserved infra namespaces are blacklisted by dedicated-admin using the “openshift-*” filter.
```

I say we keep this convention for now.  If we use another prefix that's fine but a convention of one isn't a great thing (i.e. this is the only operator we create that isn't already landing in an "openshift" namespace)